### PR TITLE
Add variables option into query type.

### PR DESCRIPTION
**Pull request by abMatGit**
Issue: https://github.com/graphql-cli/graphql-cli/issues/201

Did some preliminary testing and this seems to work. Not very experienced in typescript so if there's any standard ways of doing things feel free to help me out!

### CODE REVIEW

File: src/cmds/query.ts [hunk 1]
```diff
@@ -23,11 +23,16 @@ export const builder = {
     describe: 'Run all operations in order',
     type: 'boolean',
   },
+  variables: {
+    alias: 'g',
```
Comment (abMatGit) [inline src/cmds/query.ts]: `v` was taken so I took a random letter here.

Comment (kbrandwijk) [reply src/cmds/query.ts]: I think it's best to leave it out completely. We do that in other places too.

**Comment by kbrandwijk** [comment]
One other small thing: can you maybe put a try catch around the JSON parsing and show a proper error if it fails?

File: src/cmds/query.ts [hunk 2]
```diff
@@ -36,6 +41,7 @@ export async function handler(
 
   const endpoint = config.endpointsExtension.getEndpoint(argv.endpoint)
   const query = fs.readFileSync(argv.file, { encoding: 'utf8' })
+  const variables = argv.variables || "{}"
```
Comment (kbrandwijk) [inline src/cmds/query.ts]: There is a ts-lint error on this line in the CI build from the use of double quotes.

File: src/cmds/query.ts (hunk 1 above)
Comment (kbrandwijk) [reply src/cmds/query.ts]: I would take the alias out completely if it doesn't make sense.

File: src/cmds/query.ts [hunk 3]
```diff
@@ -76,6 +83,7 @@ async function runQuery(
     body: JSON.stringify({
       query,
       operationName,
+      variables: JSON.parse(variables)
```
Comment (kbrandwijk) [inline src/cmds/query.ts]: Can you put a try/catch around the JSON parsing so you can show a proper error message if it fails?

**Comment by abMatGit** [comment]
@kbrandwijk thanks for looking into this 🙏 ! 

Re: `try/catch` -- I moved it into its own function to properly isolate the JSON parsing.

**Comment by kbrandwijk** [comment]
Thank you! 🙏

**Pull request merged**